cmd/process-agent/app/events.go

```diff
@@ -0,0 +1,90 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+
+	"github.com/DataDog/datadog-agent/cmd/process-agent/flags"
+	sysconfig "github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	"github.com/DataDog/datadog-agent/pkg/process/events"
+	"github.com/DataDog/datadog-agent/pkg/process/events/model"
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// EventsCmd is a command to interact with process lifecycle events
+var EventsCmd = &cobra.Command{
+	Use:          "events",
+	Short:        "Interact with process lifecycle events. This feature is currently in alpha version and needs root privilege to run.",
+	SilenceUsage: true,
+}
+
+// EventsListenCmd is a command to listen for process lifecycle events
+var EventsListenCmd = &cobra.Command{
+	Use:          "listen",
+	Short:        "Open a session to listen for process lifecycle events. This feature is currently in alpha version and needs root privilege to run.",
+	RunE:         runEventListener,
+	SilenceUsage: true,
+}
+
+func init() {
+	EventsCmd.AddCommand(EventsListenCmd)
+}
+
+func runEventListener(cmd *cobra.Command, args []string) error {
+	ddconfig.InitSystemProbeConfig(ddconfig.Datadog)
+
+	configPath := cmd.Flag(flags.CfgPath).Value.String()
+	var sysprobePath string
+	if cmd.Flag(flags.SysProbeConfig) != nil {
+		sysprobePath = cmd.Flag(flags.SysProbeConfig).Value.String()
+	}
+
+	if err := config.LoadConfigIfExists(configPath); err != nil {
+		return log.Criticalf("Error parsing config: %s", err)
+	}
+
+	// Load system-probe.yaml file and merge it to the global Datadog config
+	sysCfg, err := sysconfig.Merge(sysprobePath)
+	if err != nil {
+		return log.Critical(err)
+	}
```

Comment:
Wonder if at some point we could consolidate this with other places where we have similar logic.

Reply:
I think that this is a really good idea. I've moved this logic to a new `bootstrapEventsCmd` function in the EventStore PR so it can be reused by all events subcommand. 
However, I believe that this could be potentially used by other process-agent commands as well. I'll wait to refactor it in a separate PR since it touches other commands if you don't mind.